superset-frontend/src/dashboard/actions/sliceEntities.js

```diff
@@ -39,92 +39,154 @@ export function fetchAllSlicesFailed(error) {
   return { type: FETCH_ALL_SLICES_FAILED, payload: { error } };
 }
 
+function fetchSlices(
+  userId,
+  excludeFilterBox,
+  dispatch,
```

Comment:
you could transform this to an action creator function (like the ones below) to avoid passing this param around